dao.py

```diff
@@ -619,6 +618,25 @@ def get_user_by_session_id_or_none(self, session_id):
     def get_user_by_region(self, regions):
         pass
 
+    def get_is_superadmin(self, user_id):
+        user = None
```

Comment:
I think it is probably better to use `get_user_by_id_or_none` here instead of rewriting the Mongo queries (so something like `user = get_user_by_id_or_none(user_id)`.

Actually, do we use this function (get_is_super_admin) anywhere else in the code? If not, I think it's fine to remove this function (it is easy enough to just check whether user.admin_level=='SUPER' whenever we have a user object).